main/resource-access.bs

```diff
@@ -5,28 +5,65 @@ Issue: Write introduction to the Authenticated Resource Access section.
 
 ## Hypertext Transfer Protocol ## {#http}
 
-Issue: Write Hypertext Transfer Protocol section.
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Message Syntax and Routing [[!RFC7230]]
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Semantics and Content [[!RFC7231]].
-
-Draft:
-A Solid data pod MUST implement HTTP/1.1 Authentication [[!RFC7235]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Conditional Requests [[!RFC7232]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Range Requests [[!RFC7233]].
-
-Draft:
-A Solid data pod SHOULD implement HTTP/1.1 Caching [[!RFC7234]].
-
-Draft:
-A Solid data pod MAY implement HTTP/2 [[!RFC7540]].
+### Background and Need ### {#http-need}
+<em>This section is non-normative.</em>
+
+Solid clients and servers need to exchange data securely over the Internet,
+and they do so using the Web protocol HTTP.
```

Comment:
`protocol HTTP.`
-> 'protocol, HTTP.`

Reply:
I intended it without actually. Like, "the Web protocol WebSockets", "the Web protocol CORS", "the Web protocol Memento". Not "_the_ Web protocol, namely HTTP".

Reply:
Ah. IME, what you meant would more commonly be presented (and understood) as "the WebSockets Web protocol", "the CORS Web protocol", etc.

Reply:
I find "the HTTP protocol" easier to read. "the HTTP Web protocol" is also okay to use in this case because of "Internet" mention, but anything after this need not mention "Web" - I think the context is obvious enough, and if need to distinguish somewhere for some reason, that can be done on a case by case basis.

Reply:
Indeed; the Internet/Web tuple was what I was going for.

Reply:
(Of course, "the HTTP protocol", and even "the HTTP Web protocol", is redundant, as it's saying "the HyperText Transfer Protocol [Web] protocol"... unlike the other examples.)

Reply:
"the HTTP Web standard". You're welcome 😉